rfcs/0000-multiple-kuadrant-instances.md

```diff
@@ -0,0 +1,152 @@
+# RFC 0000
+
+- Feature Name: `multiple kuadrant instances`
+- Start Date: 2023-01-12
+- RFC PR: [Kuadrant/architecture#0000](https://github.com/Kuadrant/architecture/pull/0000)
+- Issue tracking: [Kuadrant/architecture#0000](https://github.com/Kuadrant/architecture/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a new kuadrant architecture design to enable **multiple kuadrant instances** to be running in a single cluster.
+
+![](https://i.imgur.com/ZsPibfO.png)
+
+# Motivation
+[motivation]: #motivation
+
+The main benefit of multiple Kuadrant instances in a single cluster is that it allows dedicated Kuadrant's services for tenants.
+
+Dedicated Kuadrant deployment brings lots of benefits. Just to name a few:
+* Protection against external traffic load spikes. Other tenant's traffic spikes does not affect Authorino/Limitador throughput and delay as it would when shared.
+* No need to have cluster administrator role to deploy a kuadrant instance. One tenant administrator can manage gateways, Limitador and Authorino instances (including deployment modes).
+* The cluster administrator gets control and visibility across all the Kuadrant instances, while the tenant administrator only gets control over their specific gateway(s), Limitador and Authorino instances.
+* (looking for ideas for more benefits)...
```

Comment:
I think the key benefit is protection against noisy neighbour and isolation particularly in the auth context.